### DISCUSSION TITLE
Is Thare Any Good Cosmos Window System

### DISCUSSION THREAD

**Discussion by ghost** [CosmosOS/Cosmos#3120] [Q&A] [answered]
**So I Need A New Window System** For My Project I Kinda Need One That Has Buttons, Text, Rect, Pixels etc. Does Any One Know Of Any? 🦖

**Comment by ADev531** [comment, marked as answer]
I recommend entering CosmosOS official discord server and download a ui library.

**Reply by cabfile** [reply]
[guess what](https://www.google.com/search?q=minimum+age+to+use+github)
